src/js/components/Form/Form.js

```diff
@@ -85,15 +85,21 @@ const Form = forwardRef(
       setValue(prevValue => {
         const nextValue = { ...prevValue };
         nextValue[name] = data;
-
         // re-run any validations, in case the validation
         // is checking across fields
         setErrors(prevErrors => {
           const nextErrors = { ...prevErrors };
           Object.keys(prevErrors).forEach(errName => {
             if (validations.current[errName]) {
-              const nextError = validations.current[errName](data, nextValue);
-              updateErrors(nextErrors, errName, nextError);
+              const nextError = validations.current[errName](nextValue, data);
```

Comment:
This seems backwards, the two arguments to the validation should be `(data, nextValue)`.

Reply:
console.log of validations.current 
```
name: ƒ validateField(value2, data)
email: ƒ validateField(value2, data)
```

i could be misunderstanding something.

Reply:
Take a look at all of the other calls, such as `validations.current[name](value[name], value)`. At line 94, `nextValue` is the same kind of thing as `value` and `value[name]` is the same thing a `data` in this context.

Reply:
gotcha, thanks for clearing that up.

Reply:
changes made